src/main/java/se/sundsvall/parkingpermit/businesslogic/worker/UpdateErrandPhaseTaskWorker.java

```diff
@@ -56,4 +63,10 @@ public void executeBusinessLogic(ExternalTask externalTask, ExternalTaskService
 			failureHandler.handleException(externalTaskService, externalTask, exception.getMessage());
 		}
 	}
+
+	private boolean isErrandFinalized(Errand errand) {
```

Comment:
Why should status canceled indicate that errand is finalized?

Suggestion: Add phaseStatus as an optional process variabel that defaults to "ongoing" if variable is not set.